plugins/formal-verification/testData/diagnostics/no_contracts.fir.diag.txt

```diff
@@ -40,3 +40,15 @@ method global$pkg_$with_int_declaration() returns (ret$: Int)
   local$x := 0
   ret$ := local$x
 }
+
+/no_contracts.kt:(285,295): info: field backing_int: Int
+
+method global$pkg_$while_loop(local$b: Bool) returns (ret$: Int)
+{
+  while (local$b) {
+    var local$a: Int
+    local$a := 1
+    ret$ := local$a
```

Comment:
This is not quite right: the control flow here isn't the same.  I think it may be best to use an example where the Viper code is the one we'll want to have, so we don't have to fix tests later.

Reply:
In the Kotlin code, the `return` causes the function to exit.  However, we don't yet support this functionality of `return` in the translation: we just turn `return a` into `ret := a`, and then the loop continues.  When we add support for this, we'll have to come back and update the expected Viper output here, which may be confusing.  I think not having a `return` in the loop would be better because of this.